pkg/dns/vips/tag_first_virtual_outbound_view.go

```diff
@@ -0,0 +1,214 @@
+package vips
+
+import (
+	"reflect"
+	"strings"
+
+	"github.com/asaskevich/govalidator"
+
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
+	"github.com/kumahq/kuma/pkg/plugins/runtime/k8s/util"
```

Comment:
not a fan of bringing this dependency here.

Could we replace this with https://pkg.go.dev/golang.org/x/exp/maps#Clone ?
If not, can we move this to `pkg/util/maps`